src/control/server/storage/scm/ipmctl.go

```diff
@@ -531,16 +584,11 @@ func (cr *cmdRunner) removeNamespace(devName string) error {
 	return nil
 }
 
-// createNamespaces repeatedly creates namespaces until no free capacity.
-func (cr *cmdRunner) createNamespaces() (storage.ScmNamespaces, error) {
-	if err := cr.checkNdctl(); err != nil {
-		return nil, err
-	}
-
+func (cr *cmdRunner) createSingleNamespacePerNUMA() (storage.ScmNamespaces, error) {
 	for {
 		cr.log.Debug("creating pmem namespace")
 
-		if _, err := cr.createNamespace(); err != nil {
+		if _, err := cr.createNamespace(-1, 0); err != nil {
```

Comment:
OK, I see that `-1` is a valid value. Please define these as consts, for both readability and maintainability.

Reply:
done